LIBVERSION

```diff
@@ -1 +1 @@
-1.6.1
+2.0.0
```

Comment:
I don't know what to touch in libstorage-ng.spec.in. In that file there is `Version:        @VERSION@` which is the package version, but not the library version.

I guess libstorage-ng1-4.1.79-0.x86_64.rpm is correct name for the package. In my TW I see libstorage-ng1-4.1.75-1.1.x86_64.

@wfeldt Could you please indicate what is exactly needed to touch to increase the major version of the library? Asking you because I see that libstorage-ng is using linuxrc tools for versioning.

Reply:
Change ```%define libname %{name}1``` to  ```%define libname %{name}2```. Without that building locally fails with rpmlint complaining about a shared library policy violation:

```
libstorage-ng1.x86_64: E: shlib-policy-name-error (Badness: 10000) libstorage-ng2
Your package contains a single shared library but is not named after its
SONAME.
```

Reply:
Thanks ;)